cmd/controller/ctrl.go

```diff
@@ -31,6 +32,8 @@ import (
 	clientset "github.com/openebs/node-disk-manager/pkg/client/clientset/versioned"
 )
 
+var UdevEventMessages = make(chan map[string]string)
```

Comment:
Need to provide comments that explain this variable & its use. You may want to explain `why the use of chan` & so on. The whole point is making things simpler to understand and hence easy to extend.

Reply:
added